resources/assets/less/bem/beatmap-discussion-user-card.less

```diff
@@ -50,7 +50,7 @@
   }
 
   &__user-link {
-    color: white;
```

Comment:
more like this isn't thorough enough - it should have correct hover color as well otherwise it ends up with white text and colored underline on hover

also I don't know what purpose it serves for the avatar link.

And then might as well move the whole `__user-text` styling here and remove it (leaving just the `u-ellipsis-overflow` for the `span`)

Reply:
...or if the link isn't flex, it can have the `u-ellipsis-overflow` instead? I don't remember why it's structured like that (is it because the link originally covered the whole card?)

Reply:
> white text and colored underline on hover
which isn't a change from the current styling?

Then there's all the different combinations currently:
- this; white text, coloured underline, no card
- `mapped by x`; white text, no change on hover, has card
- links in the nomination box; coloured text, underline on hover, has card
- scoreboard; white text, colour change + underline on hover, has card
👀

Reply:
>  which isn't a change from the current styling?

I did say "maybe fix that as well while at it" in the previous comment... Text and underline having different colour doesn't make much sense.